lib/engine/game/g_system18/step/buy_train.rb

```diff
@@ -14,7 +14,7 @@ def actions(entity)
           end
 
           def setup
-            @emr_issue = false
+            @emergency_issued = false
```

Comment:
You're referencing both `@emergency_issued` and `@round.emergency_issued` in this file. These will be two different variables.